apps/src/netsim/NetSimMetronome.js

```diff
@@ -77,25 +76,19 @@ NetSimMetronome.prototype.tick = function (clock) {
   if (this.pulseIntervalMillis_ === Infinity) {
```

Comment:
i think we discussed in the past john carmack's aversion to having a magical value to indicate state, but no worries :)